scripts/env_matrix.yml

```diff
@@ -4,6 +4,7 @@ CONDA_PY:
   - "3.5"
 CONDA_BOOST: "1.60"
 CONDA_PERL: "5.22.0"
+iCONDA_R: "3.3.0"
```

Comment:
You are of course right. I was mislead by the front page of the R package on anaconda.org, which suggests 3.3.0. This seems to be due to the windows version which is already at 3.3.0.

Thanks! If it will be fixed this week, I can just keep this PR open.